pyvista/core/filters/rectilinear_grid.py

```diff
@@ -94,7 +94,7 @@ def to_tetrahedra(
         alg.SetRememberVoxelId(pass_cell_ids or pass_data)
         if mixed is not False:
             if isinstance(mixed, str):
-                self.cell_data.active_scalars_name = mixed
+                self.cell_data.active_scalars_name = mixed  # type: ignore[attr-defined]
```

Comment:
typing `self: RectilinearGrid` here might fix this error, though a separate "sweep" of this file still need to be done to add these annotations to all filters It's fine if you leave it / don't change it

Reply:
this is ok to fix later